### ISSUE TITLE
sugestões ao repositório do forum

### ISSUE THREAD

**Issue by felquis** [frontendbr/forum#1796]
- mover wiki para arquivos MD
    não tem nem como enviar pull request para uma wiki, já os arquivos MD são muito mais versáteis por serem arquivos.

- desabilitar coisas inúteis para o forum
    como, actions, projects, wiki

é isso

**Comment by vagnerlandio** [comment]
Concordo, eu nem sabia que existia Wiki aqui😅, uso o aplicativo oficial do GitHub e aqui não aparece ou não sei onde fica 🤔

**Comment by felipefialho** [comment]
Nem eu sabia que tínhamos Wiki aqui hahahaha 😨 

Boas sugestões @felquis, consegue abrir um PR movendo essa Wiki misteriosa pra um arquivo `md`?
